src/zope/app/exception/browser/systemerror.rst

```diff
@@ -12,10 +12,10 @@ terse message and that sets the response status.
 There is a simple view registered in ``ftesting.zcml`` which raises
 ``Exception()``:
 
-  >>> print http(r"""
+  >>> print(http(r"""
   ... GET /error.html HTTP/1.1
-  ... """)
-  HTTP/1.1 500 Internal Server Error
+  ... """))
+  HTTP/1.0 500 Internal Server Error
```

Comment:
One of the differences between `zope.app.testing` and `zope.app.wsgi.testlayer` is this. I've seen that on each of the projects I've updated.